unison-cli/src/Unison/Codebase/Editor/Output.hs

```diff
@@ -623,6 +624,6 @@ isNumberedFailure = \case
   ShowDiffAfterModifyBranch {} -> False
   ShowDiffAfterPull {} -> False
   ShowDiffAfterUndo {} -> False
-  ShowDiffNamespace {} -> False
+  ShowDiffNamespace _ _ _ bd -> BD.isEmpty bd
```

Comment:
I added this so that we can do round trip testing in the following way:

- Add a bunch of definitions to one namespace
- Edit those added definitions
- Add the pretty-printed definitions to another namespace
- Diff the two namespaces. They should be identical if the hashes are all the same. And if not, you'll be told which names have different hashes.

See the `transcripts-round-trip/main.md` file which uses this strategy.